include/global.fieldmap.h

```diff
@@ -73,9 +73,9 @@ struct ObjectEventTemplate
     /*0x06*/ s16 y;
     /*0x08*/ u8 elevation;
     /*0x09*/ u8 movementType;
-    /*0x0A*/ u16 movementRangeX:4;
+    /*0x0A*/ u16 movementRangeX:8; // Increased to 8 bits in expansion to allow GetItemBallIdFromTemplate to get more than 15 of an item
              u16 movementRangeY:4;
-             //u16 padding2:8;
+             //u16 padding2:4;
     /*0x0C*/ u16 trainerType;
```

Comment:
This doesn't look retro-compatible.

Reply:
I'm not sure what you mean by "retro-compatible". I think ultimately, it means you are suggesting one of the following

1) there is a better way to make this change
2) we should not make this change, `Common_EventScript_FindItem` should be capped at a quantity of 15
3) we should not make this change because a different member of the struct should be used
4) something else that I'm not thinking of

Reply:
As far as I remember, you're right about `kind`... My only concern with that one would be to check whether it ends up being used by the soon-to-be-PRed OW expansion (I think that's in one of merrp's branches?), I suspect one of these padding bytes ends up getting used, but I'm not sure which one!

Reply:
https://discord.com/channels/419213663107416084/1193434690321911828/1194358990411411556

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/3942/commits/bb192693a6b9282fda449facddd6d3132d3b7902